Adding a migration test file for the user data model

LeonRabinovich opened the pull request:
The test file contains 3 users which will be added to our DB after running `pipenv run python manage.py migrate`

The Users:
1.  **Name =** 'testUser1', **Email =** 'user1@gmail.com', **Password =**'password123',
2.  **Name =**'testUser2', **Email =**'David@gmail.com', **Password =**'Password456',
3.  **Name =**'testUser3', **Email =**'User3@gmail.com', **Password =**'User3Password'

Resolves: #28

Commit:
Adding a migration test file for the user data model

oriAdler reviewed (approved):
LGTM

VeraTs reviewed (approved):
LGTM

kobihk commented on homepage/migrations/0002_user_test_data.py:
```diff
@@ -0,0 +1,25 @@
+
+        users_test_data = [
+            ('testUser1', 'user1@gmail.com', 'password123'),
+            ("testUser2", "David@gmail.com", "Password456"),
```
please try to keep your string consistent by surround all with one apostrophe(') or two apostrophes (")

in my opinion it looks better

kobihk commented on homepage/migrations/0002_user_test_data.py:
```diff
@@ -0,0 +1,25 @@
+
+    operations = [
+        migrations.RunPython(generate_user_test_data),
+    ]
```
please add one empty line in the end of the file

Commit:
Fixing the users data

LeonRabinovich commented:
@kobihk 

- I fixed the strings and changed all of them to (') 

- regarding the empty line at the end of the file - its there but it is not shown in github (its the same for all of the files which were already commited)

i dont think github shows an empty line when you preview the file
